azure/dataprotection_backupvaults/dashboards/overview/main.tf

```diff
@@ -0,0 +1,97 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.76.0"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
+variable "lightstep_project" {
+  description = "Cloud Observability Project Name"
```

Comment:
Just noting that I like this for the description. Even though it won't match the variable name so closely as previous, I think it's a great description.